src/main/java/org/jitsi/jigasi/xmpp/CallControlComponent.java

```diff
@@ -122,6 +122,16 @@ public void start(BundleContext bundleContext)
         ConfigurationService config
             = ServiceUtils.getService(
                     bundleContext, ConfigurationService.class);
+        
+        Boolean always_trust_mode = config.getBoolean(
+                "net.java.sip.communicator.service.gui.ALWAYS_TRUST_MODE_ENABLED",false);
```

Comment:
@damencho I strongly disagree with that. The default should be secure. If someone needs to connect with TLS insecurely, he should have the option to set that.